nixos/tests/haproxy.nix

```diff
@@ -31,6 +31,8 @@ import ./make-test-python.nix ({ pkgs, ...}: {
           }];
         };
       };
+      # Enabled by default by `services.httpd`, conflict with the minimal profile
+      services.logrotate.enable = false;
```

Comment:
How did that leak into the commit? Also no mention in the commit message.

Reply:
It's not a leak, the test is currently failing because it imports `nixos/profiles/minimal` which set `services.logrotate.enable` to false while `services.httpd.enable = true` while set it to true.

That being said, it should have been split in a dedicated commit with an explanation. I will do that and instead of trying to explicitly set  `services.logrotate.enable` I will remove the usage of `nixos/profiles/minimal` to be consistent with the majority of NixOS tests.